auth/store.go

```diff
@@ -846,6 +856,15 @@ func NewAuthStore(be backend.Backend) *authStore {
 		simpleTokens: make(map[string]string),
 		revision:     0,
 	}
+	tokenDeleteFunc := func(t string) {
+		as.simpleTokensMu.Lock()
+		defer as.simpleTokensMu.Unlock()
+		if username, ok := as.simpleTokens[t]; ok {
+			plog.Errorf("deleting token %s for user %s", t, username)
```

Comment:
this shouldn't be Errorf since nothing is breaking and it can happen from ordinary usage; maybe Infof or Debugf